detection-rules/link_ipfs_phishing.yml

```
@@ -32,8 +32,12 @@ source: |
   // adding negation block for legitimate domains with ipfs in their name
   and not sender.email.domain.domain in ("shipfsl.com")
 
-  // negate any link extractions from additional recipients at ipfs.com
+  // negate ipfs.com issues
   and not any(recipients.to, .email.domain.domain == "ipfs.com")
+  and not (
+    sender.email.domain.root_domain == "ipfs.com"
```

avoid Fps where there is no domain (2 mimic failures)
```suggestion
      sender.email.domain.root_domain is not null 
      and sender.email.domain.root_domain == "ipfs.com"
```